web/config/webpack.config.dev.js

```diff
@@ -150,12 +150,12 @@ module.exports = {
           name: 'static/media/[name].[hash:8].[ext]'
         }
       },
-      // "url" loader works like "file" loader except that it embeds assets
+      // "calendarURL" loader works like "file" loader except that it embeds assets
```

Comment:
My bad, did some sketchy refactoring on other variables with the same name. Changing it back now.